CONTRIBUTING.md

```diff
@@ -6,7 +6,7 @@ This project welcomes community contributions. The contributions could be [Issue
 
 ## Open Development Workflow
 
-The Customer Architecture Team (CAT) is very active in this GitHub repository and carries out all the development work in the public. New issues, ideas or feature requests are filed in the repo and fixes/new implementations are implemented through pull requests that are linked to their respective issues. Code reviews can be followed as well to learn about the principles which the team follows in building and maintaining artefacts related to this repository..
+The Customer Architecture Team (CAT) is very active in this GitHub repository and carries out all the development work in the public. New issues, ideas or feature requests are filed in the repo and fixes/new implementations are implemented through pull requests that are linked to their respective issues. Code reviews can be followed as well to learn about the principles which the team follows in building and maintaining artefacts related to this repository.
```

Comment:
Consider rephrasing? 

"The Customer Architecture Team (CAT) is very active in this GitHub repository and carries out all the development work in the public."

Change to ... 

"The Customer Architecture Team (CAT) actively manages and contributes to this repository. "

Reply:
Changed